### ISSUE TITLE
Some documentation / usage examples

### ISSUE THREAD

**Issue by remcoros** [UblSharp/UblSharp#2]
- [ ] Provide some simple examples how to read / write documents (UblSharp).
- [ ] Provide some simple examples how to validate UBL XML documents and .NET objects (UblSharp.Validation).

'Getting started' (a few read/write/validate examples) can be in the README. More examples in the wiki or as code samples (samples project).

**Comment by fvandillen** [comment]
Would be awesome to have some documentation. Currently, I'm loading some UBL invoices which are sent to us on behalf of our clients. Is it true that I have to call the static UblDocument.Load method? 

Thanks for this libary!

**Comment by remcoros** [comment]
Yes, you can use UblDocument.Load. Or create your own XmlSerializer like this:

```
var s = new XmlSerializer(typeof(InvoiceType));
```

The types in UblSharp are meant to be used with the default XmlSerializer from .NET.

The UblDocument type and methods contain some helper functions for loading/saving and internally use a cached XmlSerializer per UBL document type.

**Comment by salk52** [comment]
Can someone provide sample how to create XML Invoice?

Thanx

**Comment by FParadox** [comment]
Hello
Friends

Someone who may have wanted Deserialize/Deserializate to XML UBL 2.0, to read the fields and load it into the dataset.


Please support her with a reading code example, in only two fields. I provide a recent XML with the XML version UBl 2.0

Thanks for your support and Library!

[B001-00000035.zip](https://github.com/UblSharp/UblSharp/files/1708478/B001-00000035.zip)

**Comment by OlaecheaEstibaliz** [comment]
Hi, 
I need to read an XML UBL and load it into the dataset. Is in this library any function to do it? 
Thanks!

**Comment by Abousamah** [comment]
how can i generate or Create xml ubl invoice and sign it by this library from my invoice class object

**Comment by Amit-Vienna** [comment]
Hi, 
Can anyone tell me the step and how to digitally signed the XML Document using UBL
